source/js/GliderRecipe.js

```diff
@@ -0,0 +1,201 @@
+class GliderRecipe extends HTMLElement {
+  constructor(){
+    super();
+    let shadow = this.attachShadow({mode: 'open'});
+  }
+
+  set data(data) {
+
+    let cleanData = {};
+    /** 
+     * Making a more organized object for getting 
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
     * Making a more organized object for getting
```